### PR TITLE
fixes bug 1371008 - Add CreateFileMappingA and a few others to the skiplist

### PR DESCRIPTION
**Pull request by peterbe**
I have to admit, I'm not sure how to complete this. Or test it. 
When I run my local processor it doesn't appear to update ES so I put in print statements in `crashstorage_base.py` to see what the signature *became*.

Here are the signatures generated:

Before: `u'CreateFileMappingA'`
After: `u'CreateFileMappingA | MsgWaitForMultipleObjects | mozilla::ipc::MessageChannel::WaitForInterruptNotify | mozilla::ipc::MessageChannel::Call | mozilla::plugins::PPluginScriptableObjectChild::CallNPN_Evaluate'`

Again...
Before: `u'shutdownhang | CreateFileMappingA'`
After: `u'shutdownhang | CreateFileMappingA | ConditionVariableFallback::wait | js::ConditionVariable::wait'`

Or, for the other one:
Before: `u'shutdownhang | RtlpWaitForCriticalSection | RtlEnterCriticalSection | ConditionVariableFallback::wait'`
After: `u'shutdownhang | RtlpWaitForCriticalSection | RtlEnterCriticalSection | ConditionVariableFallback::wait | js::ConditionVariable::wait_for'`

Is that how this is done?

### CODE REVIEW

**Comment by peterbe** [comment]
Oh now I think I understand why my ES isn't updated thru the webapp. My webapp uses the stage ES but my local processor uses the ES on `localhost:9200`. But it's besides the point. The print statements allowed me to see the difference.

File: socorro/siglists/prefix_signature_re.txt [hunk 1]
```diff
@@ -19,8 +19,10 @@ CERT_.*
 CFRelease
 _chkstk
 CleanupPerAppKey
+ConditionVariableFallback::wait.*
```
Comment (adngdb) [inline socorro/siglists/prefix_signature_re.txt]: I would do ``ConditionVariableFallback::wait(.*)`` in order to not catch unexpected things, like ``waitForSmth`` (if that ever exists).

Comment (peterbe) [inline socorro/siglists/prefix_signature_re.txt]: Doesn't the `(` need to be `\(` then?

Comment (peterbe) [inline socorro/siglists/prefix_signature_re.txt]: See [addition](https://github.com/mozilla-services/socorro/pull/3846/commits/61f7d24ca4a226035fd7326ddcadd2590c334bd4)

```python
>>> import re
>>> regex = re.compile('ConditionVariableFallback::wait\(.*\)')
>>> regex.findall('ConditionVariableFallback::wait(_RTL_CRITICAL_SECTION*, unsigned long)')
['ConditionVariableFallback::wait(_RTL_CRITICAL_SECTION*, unsigned long)']
>>> regex = re.compile('ConditionVariableFallback::wait(.*)')
>>> regex.findall('ConditionVariableFallback::wait(_RTL_CRITICAL_SECTION*, unsigned long)')
['(_RTL_CRITICAL_SECTION*, unsigned long)']
```

Perhaps it doesn't matter *what* it matches on (which is what the brackets are for) as long as it matches *anything*.

**Pull request merged**